src/slam_toolbox_multirobot.cpp

```diff
@@ -0,0 +1,254 @@
+/*
+ * multirobot_slam_toolbox
+ * Copyright Work Modifications (c) 2023, Achala Athukorala
+ *
+ * THE WORK (AS DEFINED BELOW) IS PROVIDED UNDER THE TERMS OF THIS CREATIVE
+ * COMMONS PUBLIC LICENSE ("CCPL" OR "LICENSE"). THE WORK IS PROTECTED BY
+ * COPYRIGHT AND/OR OTHER APPLICABLE LAW. ANY USE OF THE WORK OTHER THAN AS
+ * AUTHORIZED UNDER THIS LICENSE OR COPYRIGHT LAW IS PROHIBITED.
+ *
+ * BY EXERCISING ANY RIGHTS TO THE WORK PROVIDED HERE, YOU ACCEPT AND AGREE TO
+ * BE BOUND BY THE TERMS OF THIS LICENSE. THE LICENSOR GRANTS YOU THE RIGHTS
+ * CONTAINED HERE IN CONSIDERATION OF YOUR ACCEPTANCE OF SUCH TERMS AND
+ * CONDITIONS.
+ *
+ */
+
+#include "slam_toolbox/slam_toolbox_multirobot.hpp"
+
+namespace slam_toolbox
+{
+
+/*****************************************************************************/
+MultiRobotSlamToolbox::MultiRobotSlamToolbox(rclcpp::NodeOptions options)
+: SlamToolbox(options), localized_scan_topic_("/localized_scan")
+/*****************************************************************************/
+{
+    current_ns_ = this->get_namespace() + 1;
```

Comment:
`get_namespace` returns a string, I'm not sure what this intends to do

Reply:
After reviewing the file, the use of `current_ns_` and its modification to the published localized range scan should be documented / explained further